wca-guidelines.md

```diff
@@ -112,7 +112,7 @@ To be more informative, each Guideline is classified using one of the following
 
 - 7d+) [ADDITION] The temperature of the competition area should be 21 to 25 degrees Celsius.
 - 7f1a+) [CLARIFICATION] A full-size mat should have minimum dimensions of 30cm (left to right) by 25cm (front to back).
-- 7h2+) [ADDITION] The competitors in the competitors area should not be able to see the puzzles of the competitors on stage.
+- 7h2+) [ADDITION] The competitors in the Competitors Area should not be able to see the puzzles of the competitors on stage.
```

Comment:
Should this go capitalized?

Reply:
It should for consistency. All 6 times are capitalized in the Regs.